database/negodi.sql

```diff
@@ -22,169 +22,197 @@ CREATE TABLE `users`
-  `question42` varchar(45) DEFAULT NULL,
-  `question43` varchar(150) DEFAULT NULL,
-  `question44` varchar(45) DEFAULT NULL,
-  `question45a` varchar(45) DEFAULT NULL,
-  `question45aa` varchar(45) DEFAULT NULL,
-  `question45b` varchar(45) DEFAULT NULL,
+  `question38a` varchar
+(45) DEFAULT NULL,
+  `question38b` varchar
+(45) DEFAULT NULL,
+  `question38c` varchar
+(45) DEFAULT NULL,
+  `question39a` int
+(11) DEFAULT NULL,
+  `question39b` int
+(11) DEFAULT NULL,
+  `question40` varchar
+(45) DEFAULT NULL,
+  `question40a` varchar
+(45) DEFAULT NULL,
+  `question41` varchar
+(45) DEFAULT NULL,
+  `question42` varchar
+(45) DEFAULT NULL,
+  `question43` varchar
+(150) DEFAULT NULL,
+  `question44` varchar
+(45) DEFAULT NULL,
+  `question45a` varchar
+(45) DEFAULT NULL,
+  `question45aa` varchar
+(45) DEFAULT NULL,
+  `question45b` varchar
+(45) DEFAULT NULL,
   `question45c` date DEFAULT NULL,
-  `question45d` varchar(45) DEFAULT NULL,
-  `question46` varchar(45) DEFAULT NULL,
-  `question47` varchar(45) DEFAULT NULL,
-  `question47a` varchar(45) DEFAULT NULL,
-  `question48a` int(11) DEFAULT NULL,
-  `question48b` int(11) DEFAULT NULL,
-  `question48c` varchar(45) DEFAULT NULL,
-  `question49a` varchar(45) DEFAULT NULL,
-  `question49b` varchar(45) DEFAULT NULL,
-  `question49c` varchar(45) DEFAULT NULL,
-  `question49d` varchar(45) DEFAULT NULL,
-  `question50` varchar(45) DEFAULT NULL,
-  `question51` varchar(45) DEFAULT NULL,
-  `question52` varchar(45) DEFAULT NULL,
-  PRIMARY KEY (`id`)
-);
-
-
-
-
----- FORM ASSURANCE-partners ------
-
-CREATE TABLE `form_banque_partners`
-(
-  `id` int unsigned NOT NULL AUTO_INCREMENT,
+  `question45d` varchar
+(45) DEFAULT NULL,
+  `question46` varchar
+(45) DEFAULT NULL,
+  `question47` varchar
+(45) DEFAULT NULL,
+  `question47a` varchar
+(45) DEFAULT NULL,
+  `question48a` int
+(11) DEFAULT NULL,
+  `question48b` int
+(11) DEFAULT NULL,
+  `question48c` varchar
+(45) DEFAULT NULL,
+  `question49a` varchar
+(45) DEFAULT NULL,
+  `question49b` varchar
+(45) DEFAULT NULL,
+  `question49c` varchar
+(45) DEFAULT NULL,
+  `question49d` varchar
+(45) DEFAULT NULL,
+  `question50` varchar
+(45) DEFAULT NULL,
+  `question51` varchar
+(45) DEFAULT NULL,
+  `question52` varchar
+(45) DEFAULT NULL,
   PRIMARY KEY
-(`id`),
-  CONSTRAINT `fk_id_form_banque` FOREIGN KEY
-(`id`) REFERENCES `form_banque`
-(`id`),
-  CONSTRAINT `fk_id_form_partners` FOREIGN KEY
-(`id`) REFERENCES `partners`
-(`idpartners`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
-
----- PARTNERS ------
-
-CREATE TABLE `partners`
-(
-  `idpartners` int unsigned NOT NULL AUTO_INCREMENT,
-  `name` varchar
-(45) NOT NULL,
-  `isBanque` tinyint
-(1) DEFAULT NULL,
-  `isAssurance` tinyint
-(1) DEFAULT NULL,
-  PRIMARY KEY
-(`idpartners`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+(`id`)
+);
 
 
 --------------- QUERY -----------------
 
-user--------
+smaple user--------
```

Comment:
- "Sample" :p 
- If you want this line to be a comment, it must *begin* with "--"